tests/UrlTest.php

```diff
@@ -1,171 +0,0 @@
-<?php
-
-namespace Spatie\Crawler\Test;
-
-use Spatie\Crawler\Url;
-
-class UrlTest extends TestCase
```

Comment:
This is removed because the entire Uri is used from Guzzle, no need to test it ourself